android/app/src/main/kotlin/coop/polypoly/polypod/FeatureContainer.kt

```diff
@@ -137,12 +139,25 @@ class FeatureContainer(context: Context, attrs: AttributeSet? = null) :
                 view: WebView,
                 request: WebResourceRequest
             ): WebResourceResponse {
-                if (request.url.lastPathSegment == "favicon.ico")
+                if (request.url.lastPathSegment == "favicon.ico") {
                     return WebResourceResponse(
                         null,
                         null,
                         null
                     )
+                }
+                val rawPath = request.url.path?.removePrefix(PolyOut.fsPrefix)!!
+                if (!rawPath.startsWith("/features/") &&
+                    !rawPath.startsWith("/assets/") &&
+                    !rawPath.startsWith("/container/")
```

Comment:
How about introducing `/files` so save files into? I introduced something similar on iOS the other day, to avoid this kind of confusion. I believe it's still not too late to switch to a dedicated subfolder without having to bother with implementing a migration, and it'd basically avoid these shenanigans! :D